Parsers/ASimNetworkSession/ProductParsers/vimNetworkSessionMicrosoft365Defender.yaml

```diff
@@ -15,8 +15,43 @@ References:
 Description: |
   This Query Parser maps M365 Defender network events to the Advanced SIEM Information Model Network Session schema.
 ParserName: vimNetworkSessionMicrosoft365Defender
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_ipv4_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dstipaddr_has_any_ipv4_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dstportnumber
+    Type: int
+    Default: int(null)
+  - Name: url_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: httpuseragent_has_any
+    Type: dynamic 
+    Default: dynamic([])
+  - Name: hostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dvcaction
+    Type: dynamic
+    Default: dynamic([])
+  - Name: disabled
+    Type: bool
+    Default: false
 ParserQuery: |
- let M365Defender=(){
+  let M365Defender=(starttime:datetime=datetime(null), endtime:datetime=datetime(null)
```

Comment:
Style-wise, and since people will copy this to create their parsers, I prefer one parameter per line.

Reply:
ok